ruby_on_rails/rspec.md

```diff
@@ -38,6 +38,8 @@ You should know exactly why you are adding each one of them and why is necessary
     add_filter 'app/mailers/application_mailer.rb'
     add_filter 'app/models/application_record.rb'
     add_filter '.semaphore-cache'
+    enable_coverage :branch
+    primary_coverage :branch
   end
   SimpleCov.minimum_coverage 100
```

Comment:
```suggestion
  SimpleCov.minimum_coverage line: 100, branch: 100
```

This is how you configure branch coverage.

Reply:
Interesting, this is never mentioned in the simplecov docs

Reply:
https://github.com/simplecov-ruby/simplecov#minimum-coverage